sdk/lib/3rdparty/freetype/src/type1/t1gload.c

```diff
@@ -245,8 +243,8 @@
     if ( error )
 #ifdef __REACTOS__
     {
-        free(&decoder);
-        return error;
+      free(&decoder);
```

Comment:
You can keep the original indentation (to reduce the diff)

Reply:
OK.